modules/command/src/main/java/org/apache/fluo/command/FluoInit.java

```diff
@@ -134,65 +135,60 @@ public static void main(String[] args) {
 
     try (FluoAdminImpl admin = new FluoAdminImpl(config)) {
 
-      if (admin.applicationRunning()) {
-        System.err.println("Error - The Fluo '" + config.getApplicationName() + "' application"
-            + " is already running and must be stopped before running 'fluo init'. "
-            + " Aborted initialization.");
-        System.exit(-1);
-      }
-
       FluoAdmin.InitializationOptions initOpts = new FluoAdmin.InitializationOptions();
 
       if (opts.getUpdate()) {
         System.out.println("Updating configuration for the Fluo '" + config.getApplicationName()
             + "' application in Zookeeper using " + opts.getAppPropsPath());
-        admin.updateSharedConfig();
-        System.out.println("Update is complete.");
-        System.exit(0);
-      }
-
-      if (opts.getForce()) {
-        initOpts.setClearZookeeper(true).setClearTable(true);
+        initOpts.setUpdateWithoutInitializing(true);
```

Comment:
Two alternatives that may make this cleaner

1. We call `admin.initialize(initOpts); System.exit(0)` This would remove the need for surrounding everything else in an else block.
2. Create a `FluoAdminImpl.updateSharedConfigForStoppedApp()` method. This would first check if the app is runnung, if it is it would throw an exception, if not it would call `FluoAdminImpl.updateSharedConfig()`. With this we wouldn't have to add `updateWithoutInitializing` property to `InitializationOptions`

Reply:
If ` admin.updateSharedConfig()` fails when the application is running, then this code can still call `updateSharedConfig()`.

Reply:
I don't see any explicit check in `updateSharedConfig()` to see if the application is already running. I ran the following test in `FluoAdminImplIT` to see
```
@Test
  public void test() throws Exception {

    FluoConfiguration localConfig = new FluoConfiguration(config);
    localConfig.setProperty("fluo.test123", "${fluo.connection.application.name}");
    Assert.assertEquals(localConfig.getApplicationName(), localConfig.getString("fluo.test123"));

    try (FluoAdmin admin = new FluoAdminImpl(localConfig)) {

      InitializationOptions opts =
          new InitializationOptions().setClearZookeeper(true).setClearTable(true);
      // admin.initialize(opts);
      Thread.sleep(20000);

      admin.updateSharedConfig();
    }
  }
```
With `admin.initialize(opts);` commented out it passes, but with that line not commented out it fails due to an `AlreadyInitializedException` (from within `admin.initialize()`).

I can add an `applicationRunning()` check to the beginning of `updateSharedConfiguration()`. Do we want to limit calling that method on only stopped applications?

Reply:
> I can add an applicationRunning() check to the beginning of updateSharedConfiguration(). Do we want to limit calling that method on only stopped applications?

Yes, I think that makes sense because it avoids the issue of workers running with different config.